bot/exts/events/advent_of_code/_cog.py

```diff
@@ -26,6 +29,9 @@
 # They aren't spammy and foster discussion
 AOC_WHITELIST = AOC_WHITELIST_RESTRICTED + (Channels.advent_of_code,)
 
+MOVED_COMMAND = f"{PYTHON_PREFIX}subscribe"
+MOVED_HELP = f"NOTE: This command has been moved to {MOVED_COMMAND}"
+
```

Comment:
No need for these to be variables, as they're only used in one place now

Reply:
https://github.com/python-discord/sir-lancebot/pull/972/commits/59b2993c51d93b2af7d48f386696eceb3379d1fa